src/Text/Pandoc/Readers/Textile.hs

```diff
@@ -535,11 +536,17 @@ link = try $ do
 image :: Parser [Char] ParserState Inlines
 image = try $ do
   char '!' >> notFollowedBy space
-  _ <- attributes -- ignore for now, until we have image attributes
+  (ident, cls, kvs) <- attributes
+  let getAtt k styles = case pickStyleAttrProps [k] styles of
+                          Just v  -> [(k, v)]
+                          Nothing -> []
+  let attr = case lookup "style" kvs of
+               Just stls -> (ident, cls, getAtt "width" stls ++ getAtt "height" stls)
```

Comment:
I thought GHC was smart enough to not evaluate the same expression twice as long as it's pure?